src/fsharp/CheckDeclarations.fsi

```diff
@@ -12,62 +12,67 @@ open FSharp.Compiler.TcGlobals
 open FSharp.Compiler.Text
 open FSharp.Compiler.TypedTree
 
-val AddLocalRootModuleOrNamespace : TcResultsSink -> TcGlobals -> ImportMap -> range -> TcEnv -> ModuleOrNamespaceType -> TcEnv
+val AddLocalRootModuleOrNamespace:
+    TcResultsSink -> TcGlobals -> ImportMap -> range -> TcEnv -> ModuleOrNamespaceType -> TcEnv
 
 val CreateInitialTcEnv:
-    TcGlobals *
-    ImportMap *
-    range *
-    assemblyName: string *
-    (CcuThunk * string list * string list) list
-        -> OpenDeclaration list * TcEnv 
+    TcGlobals * ImportMap * range * assemblyName: string * (CcuThunk * string list * string list) list ->
+    OpenDeclaration list * TcEnv
 
 val AddCcuToTcEnv:
-    TcGlobals *
-    ImportMap *
-    range *
-    TcEnv *
-    assemblyName: string *
-    ccu: CcuThunk *
-    autoOpens: string list *
-    internalsVisibleToAttributes: string list
-        -> OpenDeclaration list * TcEnv
+    TcGlobals
```

Comment:
I dislike this format ... my preference is that the * trails the argument, so that it indicates there is more to come.  with the argument starting with the * indicates it's connected to the previous line ... I find it wierd.

Reply:
I agree, there's an active discussion on this in fantomas and F# style guide, and we plan to make a change.  I'll dig out a link.  

I don't really mind if we integrate this before rather than later - though if you want to wait for the fix we can

Reply:
Here's the link: https://github.com/fsprojects/fantomas/issues/1955

Reply:
@dsyme, I really really do want to wait for the fix.  Every time I tried to review this, I got to the tuples, and rage quit ... cos I'm emotional and rubbiusg :-)

Reply:
@KevinRansom That's fine :)

Reply:
This has been updated to the latest Fantomas now and this is fixed